src/__tests__/ExpoClient-test.ts

```diff
@@ -6,21 +6,6 @@ afterEach(() => {
   (fetch as any).reset();
 });
 
-test('limits the number of concurrent requests', async () => {
```

Comment:
The addition of promiseRetry means that the promiseLimit is no longer the first awaited call, so this test is no longer reliable.